modules/checklist/module.php

```diff
@@ -110,6 +110,11 @@ public function set_step_progress( $step_id, $step_progress ) : void {
 		$this->update_user_progress_in_db();
 	}
 
+	public function set_user_progress() : void {
```

Comment:
The name of the function suggests something else than what happens here
I would have it named update_user_progress, and have it accept key => value to update.